src/components/TaskForm/TaskForm.css

```diff
@@ -0,0 +1,34 @@
+.form input {
+    border: none;
+    border-bottom: 1px solid rgba(128, 128, 128, 1);
+    padding: 5px;
+    margin-top: 32px;
+    margin-bottom: 32px;
+}
+
+.form input::placeholder {
+    color: rgba(128, 128, 128, 1);
```

Comment:
i would recommend to decide which method to use : rgba, color: white or #ffffff and use it everywhere